realm/realm-library/src/androidTest/java/io/realm/LinkingObjectsManagedTests.java

```diff
@@ -592,6 +593,7 @@ public void migration_backlinkedSourceClassDoesntExist() throws IOException {
      * validate its table.  If we have been living clean lives, though, the validator for
      * `BacklinksMissingFieldTarget` should notice that there is no field named `BacklinksMissingFieldSource.xxxchild`.
      */
+    @Ignore("Need to rebuild the test library")
```

Comment:
As I think I mentioned, elsewhere, this probably won't work.  I need the exact same class to have two different definitions.  I think the only way to do that is with a separate compilation unit.  The test library needs to be rebuilt.  I'll be doing that before the commit.